### DISCUSSION TITLE
Startup delay saved setting

### DISCUSSION THREAD

**Discussion by EdwardKiefer** [Rem0o/FanControl.Releases#938] [General]
V1.19 if you change startup delay 30sec >15sec and save config the Task scheduler doesn't update to the new value.
But if you edit startup delay and then toggle start with windows it will then make proper changes to delay.

**Comment by Rem0o** [comment]
Just tested a live change... and it works?

![image](https://user-images.githubusercontent.com/14118956/178540472-f154672d-7ba2-4f59-bbc5-8bbc61e6e9bd.png)

**Reply by EdwardKiefer** [reply]
Not here, if i just change the delay nothing happens in Task scheduler.
I am on the latest win10 build.

**Reply by Rem0o** [reply]
Make sure you are not in the task interface when you make that change.

**Reply by EdwardKiefer** [reply]
I close Task scheduler and reopen after I make changes.

It was working when I first installed it and didn't need to toggle the start with windows. I had it set to something around 15sec.
I opened yesterday and noticed delay was reset to 30sec and that when i noticed the issue.

Also I am Admin user account if that matters.

W10 19044.1806

**Reply by EdwardKiefer** [reply]
Ok, I tried something from different angle, I tried to modify the delay in fan control Task scheduler trigger and it errors when I try to change the value and hit ok. even if I use one of the cosen dropdown values (30 seconds).
The error states "The task XML contains a value which is incorrectly formatted or out of range"

**Reply by EdwardKiefer** [reply]
)o, fixed the issue by changing the Security option "When running the task, use the following user account"
Change group to "users".
Now I can edit the delay in the task scheduler and Fan controller changes the delay too without toggling "start with windows".
